modules/backend/lang/en/lang.php

```diff
@@ -120,6 +120,7 @@
             'event_log' => 'Event log',
             'request_log' => 'Request log',
             'app_birthday' => 'Online since',
+            'debug_warning' => 'Debug mode is enabled! It\'s recommended to disable it on production.'
```

Comment:
@Samuell1 This might read better:
```suggestion
            'debug_warning' => 'Debug mode is enabled. This is not recommended for production installations.'
```